MaxMind.Db.SourceGenerator/AnalyzerReleases.Unshipped.md

```
@@ -0,0 +1,14 @@
+; This file lists analyzer rules that have not been released yet.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)